templates/experiments/components/experiment_version_details_button.html

```diff
@@ -0,0 +1,19 @@
+<button class="btn btn-sm btn-outline btn-primary"
+        hx-get="{% url 'experiments:experiment-version-details' team_slug=team.slug experiment_id=record.working_version_id version_number=record.version_number %}"
+        hx-target="#details_modal_content"
+        hx-trigger="click"
+        onclick="document.querySelector('#details_modal').showModal();">
+  View Details
+</button>
+
+<dialog id="details_modal" class="modal">
```

Comment:
can this be removed form here and we use the one in `experiment_version_table.html`?

Reply:
https://github.com/dimagi/open-chat-studio/pull/675/commits/11abeb5d3c12d9c1b925e5eb4c0943ce31da1d2c